configure.ac

```diff
@@ -1559,6 +1559,9 @@ if test "x$use_coverage" = "xyes"; then
   dnl Add the special gcc flags
   CFLAGS="$CFLAGS -fprofile-arcs -ftest-coverage"
   LDFLAGS="$LDFLAGS -lgcov"
+  AM_CONDITIONAL([ENABLE_COVERAGE], true)
+else
+  AM_CONDITIONAL([ENABLE_COVERAGE], false)
```

Comment:
It allows us to conditionally include gcov-stub.c in tests/unit/Makefile.am for file_writer_test. That test was giving me trouble when building locally since my code had coverage enabled and would then conflict with gcov-stub.c. I think gcov-stub.c is needed for the compile_and_unit_test step in travis-scripts/script.sh where nothing is built with coverage enabled (due to .travis.yml having JOB_TYPE=compile_and_unit_test COVERAGE=no).